### PR TITLE
17347 - Enable correction on IA

### PR DESCRIPTION
**Pull request by Jxio**
*Issue #:* [/bcgov/entity17347
](https://github.com/bcgov/entity/issues/17347)
*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-542-709ycgb8.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-542-709ycgb8.web.app

File: src/components/Dashboard/FilingHistoryList/HeaderActions.vue [hunk 1]
```diff
@@ -125,7 +125,8 @@ export default class HeaderActions extends Mixins(AllowableActionsMixin) {
       !EnumUtilities.isStatusCompleted(this.filing) &&
       !EnumUtilities.isStatusCorrected(this.filing)
     )
-    conditions[3] = () => (EnumUtilities.isTypeIncorporationApplication(this.filing) && !this.isBenBcCccUlc)
+    conditions[3] = () => (EnumUtilities.isTypeIncorporationApplication(this.filing) &&
+      !this.isBenBcCccUlc && !this.isCoop)
```
Comment (severinbeauvais) [inline src/components/Dashboard/FilingHistoryList/HeaderActions.vue:129]: Please update the unit tests in tests/unit/FilingHistoryList1.spec.ts.

**Review by severinbeauvais — APPROVED** [review]
Looks good. I'm approving on the assumption that you update the unit tests. Thanks.

**Review by JazzarKarim — APPROVED** [review]
LGTM 👍

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-filings-dev--pr-542-709ycgb8.web.app

**Comment by JazzarKarim** [comment]
Jia, there's one code smell. Not sure if it's fixable for you.

**Comment by Jxio** [comment]
> Jia, there's one code smell. Not sure if it's fixable for you.

it ask me to add at least one assertion, but I already have 12 expect(..) on unit test, it might be due to a misconfiguration or misunderstanding of the code by SonarCloud.

**Comment by severinbeauvais** [comment]
> it ask me to add at least one assertion, but I already have 12 expect(..) on unit test, it might be due to a misconfiguration or misunderstanding of the code by SonarCloud.

If it's a SC issue and it still happens after your change, we can tell SC to ignore this one.

**Comment by Jxio** [comment]
> > it ask me to add at least one assertion, but I already have 12 expect(..) on unit test, it might be due to a misconfiguration or misunderstanding of the code by SonarCloud.
> 
> If it's a SC issue and it still happens after your change, we can tell SC to ignore this one.

yes I think it's SC issue, I added two expect assertions, but still happens

**Pull request merged**